controls/radspreadsheet/howto/customize-selection.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Customize Selection
+page_title: Customize Selection
+description: Customize Selection
+slug: radspreadsheet-howto-customize-selection
+tags: customize,selection
+published: True
+position: 0
+---
+
+# Customize Selection
+
+RadSpreadsheet exposes several properties that allow you control the way the selection in the control is rendered. This article will describe the available properties and show you how to work with them.
+
+## 
```

Comment:
Should we have a header here? 
Do we want to display the separator line?

If there is no reason for this heading to be there, remove it.